src/Yoti.Auth/Activity.cs

```diff
@@ -7,15 +7,16 @@
 using Newtonsoft.Json.Linq;
 using Org.BouncyCastle.Crypto;
 using Yoti.Auth.Anchors;
+using Yoti.Auth.CustomAttributes;
 using Yoti.Auth.DataObjects;
 using static Yoti.Auth.YotiAttributeValue;
 
 namespace Yoti.Auth
 {
     internal class Activity
     {
-        private YotiUserProfile _yotiUserProfile;
-        private YotiProfile _yotiProfile;
+        private readonly YotiUserProfile _yotiUserProfile;
```

Comment:
Some comments here will be good to know which one is the old profile.

Reply:
Visual Studio underlines the deprecated methods, and shows this message when you hover over `YotiUserProfile`: ![image](https://user-images.githubusercontent.com/1381991/46952639-59c25000-d083-11e8-8093-3b1acbdd7438.png)

This will happen any place that YotiUserProfile, or the fields within it are used. Do you think it's still necessary to specify this is old, given this?